doc/rst/source/cookbook/features.rst

```diff
@@ -1128,45 +1128,88 @@ Master (dynamic) CPTs
-while :doc:`/grd2cpt` can derive the range from one or more grids.
+of an incoming CPT.  Here, :doc:`/makecpt` can be told the data range or compute
+it from data tables while :doc:`/grd2cpt` can derive the range from one or more grids.
 
 .. figure:: /_images/GMT_hinge.*
    :width: 500 px
    :align: center
 
-   The top color bar is a dynamic master CPT (here, globe) with a hinge at sea level and
+   The top color bar is a dynamic master CPT (here, globe) with a hard hinge at sea level and
    a natural range from -10,000 to +10,000 meters. However, our data range
    is asymmetrical, going from -8,000 meter depths up to +3,000 meter elevations.
    Because of the hinge, the two sides of the CPT will be stretched separately
    to honor the desired range while utilizing the full color range.
 
+All CPT master tables can be found in Chapter :ref:`Of Colors and Color Legends`
+where those with hard or soft hinges are identified by triangles at their hinges.
+
+CPTs from color lists
+~~~~~~~~~~~~~~~~~~~~~
+
+GMT can build color tables "on the fly" from a comma-separated list of colors
+and a range of *z*-values to go with them.  As illustrated below, there are
+four different ways to create such CPTs. In this example, we will operate with
+a list of three colors: red,yellow and purple, given to modules with the option **-C**\ red,yellow,purple,
+and utilize a fixed data range of *z = 0-6*.
+Four different CPTs result because we either select a *continuous* or *discrete table*, and because the *z*-intervals are
+either *equidistant* or *arbitrary*.  The top continuous color table with equidistant spacing (a) is selected
+with the range **-T**\ 0/6, meaning the colors will continuously change from red (at *z = 0*) via
+yellow (at *z = 3*) to purple (at *z = 6*). Next, a discrete table with the same range (b)
+is obtained with **-T**\ 0/6/2, yielding colors that are either constant red (*z = 0-2*), yellow (*z = 2-4*)
+or purple (*z = 4-6*). The next discrete table (c) illustrates how to specify arbitrary
+node points in the CPT by providing a comma-separated list of values (**-T**\ 0,4,5.5,6). Now, the constant
+color intervals have unequal ranges, illustrated by red (*z = 0-4*), yellow (*z = 4-5.5*) and purple (*z = 5.5-6*).  Finally, we
+create a continuous color table (d) with arbitrary nodes by giving **-T**\ 0,2,6 and adding **-Z**;
+the latter option forces a continuous CPT pinned to a given list of node values.  Now, the colors
+continuously change from red (at *z = 0*) via yellow (at *z = 2*) to purple (at *z = 6*).
+Modules that obtain the *z*-range indirectly (e.g., :doc:`/grdimage`) may use the exact data range
+to set the quivalent of a **-T**\ *min/max* option.  You may append **+i**\ *dz* to the
+color list to have the *min* and *max* values rounded down and up to nearest multiple of *dz*, respectively.
+
+.. figure:: /_images/GMT_colorlist.*
```

Comment:
It would be more clear to put the labels (a)-(d) on the left, and have the options (e.g. `-Cred,yellow,purple -T0/6`) on the right.

Reply:
Done.